Providers/Scripts/2.4x-2.5x/Scripts/Tests/test_nxOMSAutomationWorker.py

```diff
@@ -228,28 +228,29 @@ def test_get_stray_worker_and_manager_pids(self):
         stray_processes = nxOMSAutomationWorker.get_stray_worker_and_manager_wsids(processes, self.workspace_id)
         self.assertFalse(stray_processes)
 
-    def test_is_any_1_3_process_running(self):
```

Comment:
Reading more closely, can we make a method that checks all old versions and the current new version?